hello-world/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hello-world",
```

Comment:
@psinha20 You forgot to entry body-parser package into package.json file and you're using that package. It gives an error in my console. 

    throw err;
    ^

> Error: Cannot find module 'body-parser'
>     at Function.Module._resolveFilename (module.js:485:15)
>     at Function.Module._load (module.js:437:25)
>     at Module.require (module.js:513:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/home/shubham/Desktop/node-training/prakhar/Node-Assignment/hello-world/index.js:2:18)
>     at Module._compile (module.js:569:30)
>     at Object.Module._extensions..js (module.js:580:10)
>     at Module.load (module.js:503:32)
>     at tryModuleLoad (module.js:466:12)
>     at Function.Module._load (module.js:458:3)
>

Reply:
@psinha20 Also, same error for the multer and node-rest-client package. As per the assignment description, I think you don't need to use the multer and node-rest-client package. Think about it!!

**Don't repeat the same mistake again. (you had done the same mistake in previous assignment)**

cc @ketanTechracers

Reply:
@psinha20 New Errors - 

Are you sure that importing cookieParser is necessary here?

> 
> app.use(cookieParser());
>         ^
> 
> ReferenceError: cookieParser is not defined
>     at Object.<anonymous> (/home/shubham/Desktop/node-training/prakhar/Node-Assignment/hello-world/index.js:8:9)
>     at Module._compile (module.js:569:30)
>     at Object.Module._extensions..js (module.js:580:10)
>     at Module.load (module.js:503:32)
>     at tryModuleLoad (module.js:466:12)
>     at Function.Module._load (module.js:458:3)
>     at Function.Module.runMain (module.js:605:10)
>     at startup (bootstrap_node.js:158:16)
>     at bootstrap_node.js:575:3
>